deploy/README.md

```diff
@@ -0,0 +1,26 @@
+# Clientcomm deploy configuration
+
+The configuration in this folder will help us deploy clientcomm in a more
+
+## credentials checklist
+All credentials should be exported as environment variables.
```

Comment:
maybe recommend a method for exporting environment variables on execution, like `env $(cat .env) terraform plan` (assuming that'll work)